docs/pages/es/getting-started/news.md

```diff
@@ -1,29 +1,30 @@
----
-title: Noticias
-permalink: es/news/
-language: es
-excluded_in_search: true
----
-
-# News
-
-<p>Subscribe with <a href="{{ site.baseurl }}/feed.xml">RSS</a> to keep up with the latest news.
-For site changes, see the <a href="https://github.com/{{ site.github_user }}/{{ site.github_repo }}/blob/master/CHANGELOG.md">changelog</a> kept with the code base.</p>
-
-<br>
-
-{% for post in site.posts limit:10 %}
-
-   <div class="post-preview">
-   <h2><a href="{{ site.baseurl }}{{ post.url }}">{{ post.title }}</a></h2>
-   <span class="post-date">{{ post.date | date: "%B %d, %Y" }}</span><br>
-   {% if post.badges %}{% for badge in post.badges %}<span class="badge badge-{{ badge.type }}">{{ badge.tag }}</span>{% endfor %}{% endif %}
-   {{ post.content | split:'<!--more-->' | first }}
-   {% if post.content contains '<!--more-->' %}
-      <a href="{{ site.baseurl }}{{ post.url }}">read more</a>
-   {% endif %}

```

Comment:
@varunsaral its difficult to confirm that only the redirect_from has changed when the entire file contents are shown as changed. Given the size of this PR, a review becomes untenable. Will you you please se that diff changes are specific to the actual line where the change is made?

Reply:
Okay , i will do it manually this time . I tried to automate it with python as the number of files are large and that script read the whole file make changes(add tag if not present)  and rewrite the whole file. Just a quick question should i open a different pull request , as i'm going to fetch the upstream from main branch and make changes again .

Reply:
Only editing 3 files solves the bug , issue was redirect_from with slash on .html file creates issue . But i also have changed the files manually on my local branch so it will be easy to review . Should i delete this PR and make a new one?

Reply:
yes, or update this existing PR. Either way will work.

The trailing slash doesn't affect .md files, huh?